### ISSUE TITLE
Intercept Inbound & outbound SIP Packets

### ISSUE THREAD

**Issue by harisqurashi** [BelledonneCommunications/linphone-android#1467]
We simply want to intercept/process incoming & outgoing SIP packets to retrieve some custom parameters/headers inside our code to perform some additional information, rebuild the SIP packet again and send it back to our server e.g.  after successful `REGISTER`  flow upon `200 OK` we need to retrieve some custom headers like `Service-Route`, `P-Associated-URI` & `Path` and append these values on our `INVITE` request to tell our SIP server to use route mentioned in `Service-Route` along with the `P-Associated-URI`. Same flow is being used on subsequent requests. 

Long story short, is there any listener or observer is available to receive bytes inside our code?

**Comment by harisqurashi** [comment]
Why this issue is closed on both repos without sharing any information of question asked?

**Comment by harisqurashi** [comment]
Why this issue is closed on both repos without sharing any information of question asked?

**Comment by Viish** [comment]
The issue in linphone-sdk repository isn't closed...
